micro-ui/web/micro-ui-internals/packages/modules/hrms/src/components/InboxFilter.js

```
@@ -162,16 +162,17 @@ const Filter = ({ searchParams, onFilterChange, onSearch, removeParam, ...props
             )}
           </div>
           <div>
-            <div>
+             <div>
               <div className="filter-label">{t("HR_ULB_LABEL")}</div>
               <Dropdown
                 option={[...getCityThatUserhasAccess(tenantIds)?.sort((x, y) => x?.name?.localeCompare(y?.name)).map(city => { return { ...city, i18text: Digit.Utils.locale.getCityLocale(city.code) } })]}
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)